cfn/README-GD.md

```diff
@@ -66,12 +70,69 @@ Make a note of the `access_key_id` and `secret_key` values, which you need when
 curl -s -X GET -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys | jq
 ```
 
-**Use the selected access_key_id in the following curl command to delete the key:**
+**Use the selected access_key_id in the following command to delete the key:**
 
 ```
 curl -X DELETE -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys/<ACCESS_KEY_ID_HERE>
 ```
 
+
+### Windows
+
+Check the version of PowerShell on your system by running PowerShell and typing in:
+
+```
+PS C:\Users\Username> $PSVersionTable.PSVersion
+```
+
+An example of output:
+
+```
+Major  Minor  Build  Revision
+-----  -----  -----  --------
+5      1      16299  98
+
+```
+
+(In this example, PowerShell 5.1 is installed.)
+
+
+#### PowerShell >= 3.0
+
+
+In the PowerShell console, please type the following commands, and then enter your Alert Logic Cloud Insight email address & password when prompted:
+
+```
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; $creds = Get-Credential -Message "Please enter your Alert Logic Cloud Insight email address and password"; $unsecureCreds = $creds.GetNetworkCredential(); $base64AuthInfo = [Convert]::ToBase64String([Text.Encoding]::ASCII.GetBytes(("{0}:{1}" -f $unsecureCreds.UserName,$unsecureCreds.Password))); Remove-Variable unsecureCreds; $AUTH = Invoke-RestMethod -Method Post -Headers @{"Authorization"=("Basic {0}" -f $base64AuthInfo)} -Uri https://api.global-services.global.alertlogic.com/aims/v1/authenticate ; Remove-Variable base64AuthInfo; $AL_ACCOUNT_ID = $AUTH.authentication.account.id; $AL_USER_ID = $AUTH.authentication.user.id; $AL_TOKEN = $AUTH.authentication.token; if (!$AL_TOKEN) { Write-Host "Authentication failure"} else { $ROLES_RESP = Invoke-RestMethod -Method Get -Headers @{"x-aims-auth-token"=$AL_TOKEN} -Uri https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/roles ; $ROLES = $ROLES_RESP.roles.name; if ($ROLES -ne "Administrator" ) { Write-Host "Your user doesn’t have Administrator role. Assigned role is '$ROLES'" } else { $ACCESS_KEY = Invoke-RestMethod -Method Post -Headers @{"x-aims-auth-token"=$AL_TOKEN} -Uri https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys ; Write-Host $ACCESS_KEY } }
+```
+An example of a successful response is:
+
+```
+@{access_key_id=712c0b413eef41f6; secret_key=1234567890b3eea8880d292fb31aa96902242a076d3d0e320cc036eb51bf25ad}
+```
+
+**Important:** If the command returns an error about not having the necessary role, please verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
```

Comment:
@Shaddakim here and for Unix instruction we refer to AIMS APIs - is that ok to do so, i.e. is the AIMS acronym known to the user?

Reply:
Customers do not know the AIMS acronym.  That will need to be changed to something different.

Reply:
Turns out, I am wrong.  CI customers apparently know AIMS, CD customers don't.  My apologies - AIMS should be OK.

Reply:
@Shaddakim thank you! Are you ok with the rest of PR?

Reply:
Go ahead and merge it.  I have some changes as to where certain items will appear in the final version of the entire Readme, but this is good for the moment.